src/OpenTelemetry.Instrumentation.StackExchangeRedis/TracerProviderBuilderExtensions.cs

```diff
@@ -92,12 +127,14 @@ public static class TracerProviderBuilderExtensions
     /// <param name="name">Optional name which is used when retrieving options.</param>
     /// <param name="connection">Optional <see cref="IConnectionMultiplexer"/> to instrument.</param>
     /// <param name="configure">Optional callback to configure options.</param>
+    /// <param name="serviceKey">Optional service key used to retrieve the <see cref="IConnectionMultiplexer"/> to instrument from the <see cref="IServiceProvider" />.</param>
     /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddRedisInstrumentation(
         this TracerProviderBuilder builder,
         string? name,
         IConnectionMultiplexer? connection,
-        Action<StackExchangeRedisInstrumentationOptions>? configure)
+        Action<StackExchangeRedisInstrumentationOptions>? configure,
+        object? serviceKey)
```

Comment:
The order of these parameters is inconsistent with the overload added above. Above we have:

* `builder`, `name`, `serviceKey`, `configure`

But here we have `serviceKey` after `configure`. I think it would make sense to have `serviceKey` before `configre` on both of them.